src/test/java/nextstep/subway/favorite/application/FavoriteServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package nextstep.subway.favorite.application;
+
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.member.domain.Member;
+import nextstep.subway.member.domain.MemberRepository;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import nextstep.subway.station.exception.NotFoundStationException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
+
+@DisplayName("즐겨찾기 서비스 단위테스트")
+@ExtendWith(MockitoExtension.class)
+class FavoriteServiceTest {
+    public final static String EMAIL = "yong2ss.com";
+    public final static String PASSWORD = "12341234";
+    public final static Integer AGE = 20;
+
+    private Station 청담역;
+    private Station 뚝섬유원지역;
+    private Member 회원;
+    private Favorite 즐겨찾기;
+
+    private FavoriteService favoriteService;
+
+    @Mock
+    private FavoriteRepository favoriteRepository;
+
+    @Mock
+    private MemberRepository memberRepository;
+
+    @Mock
+    private StationRepository stationRepository;
+
+    @BeforeEach
+    void init() {
+        favoriteService = new FavoriteService(favoriteRepository, new StationService(stationRepository), memberRepository);
+    }
```

Comment:
@InjectMocks
```suggestion
    @InjectMocks
    private FavoriteService favoriteService;
```

선언부는 이걸로 대체해줄 수 있습니다.
지금처럼 인스턴스를 생성해서 써도 무방하나 `StationService` 처럼 연쇄적으로 계속 할당하는 부분이 부담될 수 있어요.